atd-vze/src/views/Crashes/crashDataMap.js

```diff
@@ -221,41 +197,20 @@ export const createCrashDataMap = isTempRecord => {
           lookupOptions: "atd_txdot__rwy_sys_lkp",
           lookupPrefix: "rwy_sys",
         },
-        hwy_sys: {
```

Comment:
i am not finding these fields `hwy_sys` or `hwy_sfx` anymore. do away with them or bring them into the new data model?

Reply:
`hwy_sys` is being dropped. 

i think we want to include `rpt_hwy_sfx` and `rpt_sec_hwy_sfx`. i think this will be a longer conv w/ Xavier so you can ignore for now.